src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -17,6 +17,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Import Project="Microsoft.NET.Sdk.DefaultItems.props" />
     <Import Project="Microsoft.NET.SupportedTargetFrameworks.props" />
   </ImportGroup>
+
+    <!-- Workaround: https://github.com/microsoft/msbuild/issues/4948 -->
+  <PropertyGroup>
+    <ImportFrameworkWinFXTargets>false</ImportFrameworkWinFXTargets>
```

Comment:
Shouldn't this be placed in `WindowsDesktop` Sdk targets?

Reply:
I may be mistaken but I don't see a `WindowsDesktop` `.targets` file? I'm going to merge this in and if it should be placed into a different file I'd be happy to do so.

Reply:
The `Microsoft.NET.Sdk.WindowsDesktop` package in [WPF](/dotnet/wpf) repo.